lightning/config/config_test.go

```diff
@@ -580,5 +591,25 @@ func (s *configTestSuite) TestTomlPostRestore(c *C) {
 		err := cfg.LoadFromTOML([]byte(confStr))
 		c.Assert(err, IsNil)
 		c.Assert(cfg.PostRestore.Analyze, Equals, v)
+
+		b.Reset()
+		c.Assert(enc.Encode(cfg.PostRestore), IsNil)
+		c.Assert(strings.Contains(b.String(), fmt.Sprintf(`analyze = "%s"`, v.String())), IsTrue)
```

Comment:
ditto

Reply:
done in https://github.com/pingcap/tidb-lightning/pull/441/commits/2f540f8c8dfafda17bb809a4f897f8b39b9a7c52